src/olympia/bandwagon/models.py

```diff
@@ -166,13 +103,6 @@ def remove_addon(self, addon):
     def owned_by(self, user):
         return user.id == self.author_id
 
-    def can_view_stats(self, request):
```

Comment:
Do collection stats no longer exist?

Reply:
nope, and if anything called this method (or it was tested) it would have failed because the `COLLECTION_STATS_VIEW` permission is already gone.